### DISCUSSION TITLE
Query timeout setting

### DISCUSSION THREAD

**Discussion by dabricksta** [cmoog/vscode-sql-notebook#84] [Q&A]
Hey @cmoog @lorefnon @scsmithr, thanks a ton for creating this tool.

As of now, my queries timeout at just over 30s and I would like to extend this. Is this configureable? If so, where can I do so?

I appreciate all your work.

**Comment by cmoog** [comment]
Yes. This is configurable with the `SQLNotebook.queryTimeout` option in the VS Code settings.
